File: C++/First Steps in Coding - Basics of C++/Conditional Statements/Exercises/Sum Seconds.cpp
#include <iostream>
#include <cstdlib>
using namespace std;

int main() {
	int comp1, comp2, comp3, sumMin, sumSec;
	double sumTime;
	cin >> comp1; cin >> comp2; cin >> comp3;
	sumTime = comp1 + comp2 + comp3;
	sumMin = sumTime / 60; sumSec = sumTime - (sumMin * 60);
	if (sumSec < 10) {
		cout << sumMin << ":0" << sumSec;
	}
	else {
		cout << sumMin << ":" << sumSec;
	}
}

File: Python/Basics/Conditional Statements/Exercises/SumSeconds.py
from math import floor

comp1 = int(input())        #seconds
comp2 = int(input())
comp3 = int(input())


sumTime = comp1 + comp2 + comp3

minutes = floor(sumTime / 60)
seconds = round((sumTime / 60 - floor(sumTime / 60)) * 60)



output = f"{minutes}:"

if 0 <= seconds < 10:
    output += f"0{seconds}"
else:
    output += f"{seconds}"

print(output)